### ISSUE TITLE
Color and Date Field width

### ISSUE THREAD

**Issue by vitaliyb** [laravel/nova-issues#6468]
- Laravel Version: 10.40.0
- Nova Version: 4.34.3
- PHP Version: 8.2
- Browser type and version: Google Chrome 126.0.6478.183

### Description:

After udating `laravel/nova package` from `4.28` to `4.34.3`
`Laravel\Nova\Fields\Color` as well as `Laravel\Nova\Fields\Date` take up full width of the form.

I found no mentions of width change in release notes.

Before:
<img width="1268" alt="image" src="https://github.com/user-attachments/assets/651d4f91-0b9f-4c68-b3f9-cb7311add30e">

After:
<img width="1257" alt="image" src="https://github.com/user-attachments/assets/21d7a291-08c4-41b6-91d0-339a0771f751">

### Detailed steps to reproduce the issue on a fresh Nova installation:

- create resource
- add field

```php
use Laravel\Nova\Fields\Color;

class ColorField
{
    public static function make(): Color
    {
        return Color::make(__('Color'), 'color');
    }
}
```

**Comment by jeremynikolic** [comment]
Thanks for reporting this @vitaliyb 🙏
This will be addressed in Nova 5 along with other alignments for some of those fields where full width is not pertinent UI/UX wise

**Comment by crynobone** [comment]
Released with Laravel Nova 5.0.0

Feel free to open up a new issue if you're still experiencing this problem on the latest version.
